annotations/halkyon-annotations/src/main/java/io/dekorate/halkyon/adapter/ComponentConfigAdapter.java

```diff
@@ -53,8 +53,8 @@ public static ComponentConfigBuilder newBuilder(HalkyonComponent instance) {
             i.secret(),
             i.configmap(),
             i.field())).collect(Collectors.toList()).toArray(new io.dekorate.kubernetes.config.Env[0]),
-        Arrays.asList(instance.labels()).stream().map(i -> new io.dekorate.kubernetes.config.Label(i.key(),
-            i.value())).collect(Collectors.toList()).toArray(new io.dekorate.kubernetes.config.Label[0]),
+                                                                                     Arrays.asList(instance.labels()).stream().map(i -> new io.dekorate.kubernetes.config.Label(i.key(), i.value(),
+                                                                                                                                                                                i.kinds())).collect(Collectors.toList()).toArray(new io.dekorate.kubernetes.config.Label[0]),
```

Comment:
Formatting looks messed up

Reply:
Weird as we now use a maven plugin for formatiing. Will look closer.

Reply:
Formatter is not used by default. I will address this in a separate issue.